scripts/ramfs.go

```diff
@@ -398,6 +379,33 @@ func main() {
 		log.Fatal("Setup failed")
 	}
 
+	// For each arg, use it as a Glob pattern and add any matches to the
+	// package list. If there are no arguments, use [a-zA-Z]* as the glob pattern.
+	var pat []string
+	for _, v := range flag.Args() {
+		pat = append(pat, path.Join(config.Urootpath, "cmds", v))
+	}
+	if len(flag.Args()) == 0 {
+		pat = []string{path.Join(config.Urootpath, "cmds", "[a-zA-Z]*")}
+	}
+
+	debug("Initial glob is %v", pat)
+	for _, v := range pat {
+		g, err := filepath.Glob(v)
```

Comment:
we discuss and you agree we don't need to remove glob and I agree I need to add negative matches on the next PR